lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -341,6 +342,19 @@ describe('workers/repository/process/lookup/index', () => {
       expect(res.updates).toMatchSnapshot();
       expect(res.updates[0].updateType).toBe('minor');
     });
+
+    it('handles the in-range-only strategy', async () => {
+      config.currentValue = '^1.2.1';
+      config.lockedVersion = '1.2.1';
```

Comment:
```suggestion
      config.currentValue = '~1.2.0';
      config.lockedVersion = '1.2.0';
```

Let's test this way to be sure we'll get more than one response

Reply:
Hi @rarkins I have a problem when testing the way you proposed. If I keep the proposed range of ``~1.2.0` and currentValue of `1.2.1`, the lookup returns `[]`. This is correct, since the `newVersion` is `1.4.1` in this test, which is outside of range you defined. The previous `currentValue` of `^1.2.1` worked, because version `1.4.1` belongs to this range. Maybe you can suggest other range for this test?